test/module/AllowlistModule.t.sol

```diff
@@ -338,13 +341,15 @@ contract AllowlistModuleTest is CustomValidationTestBase {
             HookConfigLib.packValidationHook(address(allowlistModule), HOOK_ENTITY_ID),
             abi.encode(HOOK_ENTITY_ID, allowlistInit)
         );
-
+        // patched to also work during SMA tests by differentiating the validation
+        _signerValidation =
+            ModuleEntityLib.pack(address(singleSignerValidationModule), TEST_DEFAULT_VALIDATION_ENTITY_ID - 1);
         return (
             _signerValidation,
             true,
             true,
             new bytes4[](0),
-            abi.encode(TEST_DEFAULT_VALIDATION_ENTITY_ID, owner1),
+            abi.encode(TEST_DEFAULT_VALIDATION_ENTITY_ID - 1, owner1),
```

Comment:
Same here, could we define some other value to not have it be derived?

Reply:
Addressed this and the last one here! [a8eb220](https://github.com/erc6900/reference-implementation/pull/123/commits/a8eb22026f3a7fb1a38b1a4007a7b2a2b229a88d)